Makefile

```diff
@@ -74,7 +74,11 @@ fix: bin/golangci-lint ## Fix lint violations
 	bin/golangci-lint run --fix
 
 .PHONY: docker-image
-docker-image:
+docker-image: ## Pull base images, then build the image.
+	@sudo docker build --pull -t $(DOCKER_IMAGE) .
```

Comment:
`sudo` is not gonna fly here for sure 🙂

Reply:
Not sure what you mean by that?

Reply:
This line hasn't been changed from existing makefile, it is only moved. So `sudo` isn't an addition.

With that being said, perhaps `sudo` should be removed, even if it wasn't added by this PR.

Reply:
Sorry, missed that it was already there. :) Yeah, but I don't think this PR will happen anyway.